src/containerapp/azext_containerapp/_cloud_build_utils.py

```diff
@@ -191,10 +211,10 @@ def spin():
         thread = display_spinner("Buildpack: Initializing")
         log_execution_phase_pattern = r"===== (.*) =====$"
         current_phase_logs = ""
-        for line in response_log_streaming.iter_lines():
+        for line in response_log_streaming_lines:
             log_line = remove_ansi_characters(line.decode("utf-8"))
             current_phase_logs += f"{log_line}\n{substatus_indentation}"
-            if "ERROR:" in log_line or "Kubernetes error happened" in log_line:
+            if "----- Cloud Build failed with exit code" in log_line or "Exiting with failure status due to previous errors" in log_line:
```

Comment:
May I ask if `Cloud Build failed with exit code` and `Exiting with failure status due to previous errors` likely to change soon?

Reply:
They're not. These are errors we are generating on our side so we have control over them.